source/_includes/common-tasks/enable_i2c.md

```diff
@@ -1,23 +1,25 @@
 ## Enable I2C
 
-Home Assistant using the Home Assistant Operating System is a managed environment, which means you can't use existing methods to enable the I2C bus on a Raspberry Pi.
+Home Assistant using the Home Assistant Operating System is a managed environment, which means you can't use existing methods to enable the I2C bus on a Raspberry Pi. In order to enable I2C devices you will have to 
+- Enable I2C for the Home Assistant Operating System 
+- Enable Raspberry Hats including the I2C integration
+- Setup I2C sensors
 
-### Step by step instructions
+### Step by step instructions to enable I2C for the Home Assistant Operating System
 
-You will need:
+#### Use SD Card/Step 1 - Access the Home Assistant Operating System boot partition
 
+You will need:
 - SD card reader
 - SD card with Home Assistant Operating System flashed on it
 
-#### Step 1 - Access the Home Assistant Operating System boot partition
-
 Shutdown/turn-off your Home Assistant installation and unplug the SD card.
 Plug the SD card into an SD card reader and find a drive/file system named
 `hassos-boot`. The file system might be shown/mounted automatically. If not,
 use your operating systems disk management utility to find the SD card reader
 and make sure the first partition is available.
 
-#### Step 2 - Add files to enable I2C
+#### Use SD Card/Step 2 - Add files to enable I2C
```

Comment:
Let's not add "Use SD Card" to each header, instead, make the introduction correct.

Reply:
Removed SD card reference